client/src/featureform/serving.py

```diff
@@ -147,6 +147,24 @@ def close(self):
         """Closes the connection to the Featureform instance."""
         self.impl.close()
 
+    def iterate_feature_set(self, *features):
```

Comment:
I didnt make it a list because in the RFC simba directly enters the names of the features as arguments, instead of passing it as a list. We could change it if you think having a list is better

Reply:
I guess double check what the paramter should be, but we should still rename the function, check for at least 1 value, and make sure that we're either getting a feature object or name/variant tuple